origin-bridge/tests/services/test_attestation.py

```diff
@@ -194,6 +194,28 @@ def test_verify_phone_invalid_code():
     assert(validation_err.value.field_names[0]) == 'code'
 
 
+@mock.patch('logic.attestation_service.requests')
+@responses.activate
```

Comment:
I didn't see in other tests a way to investigate the request via `responses`, but if there's one I'd love to learn about it as it's probably better than mocking the request.

Reply:
You should be able to do something like `responses.calls[0].request` to get the original request. This is fine, but feel free to refine it if you want.